### PR TITLE
Added `prettier-plugin-nginx` to Community Plugins list

### PR DESCRIPTION
**Pull request by joedeandev**
## Description

Added `prettier-plugin-nginx` to Community Plugins list. Includes changelog and cspell update.

## Checklist

- [x] I’ve added tests to confirm my change works.
- [x] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [x] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: changelog_unreleased/markdown/13320.md [hunk 1]
```diff
@@ -0,0 +1 @@
+#### Add `prettier-plugin-nginx` to Community Plugins (#13320 by @joedeandev)
```
Comment (fisker) [inline changelog_unreleased/markdown/13320.md:1]: Normally we don't add changelog for such commit. It doesn't belong to markdown anyway.

Comment (joedeandev) [reply changelog_unreleased/markdown/13320.md]: Noted - commit amended to exclude the offending file.

**Review by fisker — APPROVED** [review]

**Comment by fisker** [comment]
BTW, we've release [Prettier@3.0.0-alpha.0](https://github.com/prettier/prettier/releases/tag/3.0.0-alpha.0), please get ready for it. 😄

**Comment by joedeandev** [comment]
> BTW, we've released [Prettier@3.0.0-alpha.0](https://github.com/prettier/prettier/releases/tag/3.0.0-alpha.0), please get ready for it. smile

Thanks for the heads-up - will do!

**Review by kachkaev — APPROVED** [review]

**Pull request merged**